.changeset/afraid-eagles-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@definitelytyped/dtslint": patch
+---
+
+Add @arethetypeswrong/cli run
```

Comment:
All of the other packages are going to need a changeset too (the tooling can't detect missing changesets, only "no changesets at all")